src/ResourceManager/Automation/Commands.Automation/StartupScripts/AzureRmAutomationStartup.ps1

```diff
@@ -14,4 +14,5 @@
 
 @{
     "Import-AzureRmAutomationModule" = "New-AzureRmAutomationModule";
-}.GetEnumerator() | Select @{Name='Name'; Expression={$_.Key}}, @{Name='Value'; Expression={$_.Value}} | Set-Alias -Description "AzureAlias" -Scope Global | Out-Null
+}.GetEnumerator() | Select @{Name='Name'; Expression={$_.Key}}, @{Name='Value'; Expression={$_.Value}} | `
+    ForEach-Object {if (!(Get-Alias -Name $_.Name -ErrorAction SilentlyContinue)) {Set-Alias -Name $_.Name -Value $_.Value -Description "AzureAlias" -Scope Global}} | Out-Null
```

Comment:
Don't need to update changelog here because this was added on Friday with a changelog update.